src/main/java/com/example/springbookstore/service/BookService.java

```diff
@@ -1,11 +1,22 @@
 package com.example.springbookstore.service;
 
-import com.example.springbookstore.entity.Book;
+import com.example.springbookstore.dto.BookDto;
+import com.example.springbookstore.dto.CreateBookRequestDto;
 import java.util.List;
 
 public interface BookService {
 
-    Book save(Book book);
+    BookDto createBook(CreateBookRequestDto createBookRequestDto);
 
-    List<Book> findAll();
+    List<BookDto> getAllBooks();
+
+    BookDto getBookById(Long id);
+
+    List<BookDto> findAll();
+
+    BookDto findById(Long id);
```

Comment:
what is the difference between getAllBooks() and findAll()?
same for getBookById() and findById()?

Please remove logical duplicates